malware/system/ksp-block-symbiote-malware-certbotx64-dnscat2.yaml

```diff
@@ -0,0 +1,40 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-block-symbiote-malware-certbotx64-dnscat2
+  namespace: default
+spec:
+  tags: ["MALWARE", "SYMBIOTE","BPF"]
+  message: "Alert! certbotx64 and dnscat2 access found! possible symbiote malware!"
+  selector:
+    matchLabels:
+      container: symbiote
+  file:
+    severity: 1
+    matchPaths:
+    - path: /etc/host.conf
+      readOnly: true
+    - path: /etc/ld.so.cache
+      readOnly: true
+    - path: /etc/resolv.conf
```

Comment:
web-server can run. But none can modify any config files. That malware is capable of modify that file to add DNS server. To prevent from that, that's why it's only `readOnly: true`.